fixtures/devtools/iframe/server.js

```diff
@@ -0,0 +1,14 @@
+const finalhandler = require('finalhandler');
```

Comment:
nit would be nice to make this file executable:
```sh
chmod +x ./fixtures/devtools/iframe/server.js
```

Then add a header:

```suggestion
#!/usr/bin/env node

const finalhandler = require('finalhandler');
```

Reply:
Done :)